pycognaize/common/utils.py

```diff
@@ -628,3 +630,39 @@ def cloud_interface_login(login_instance: Login) -> CloudInterface:
     else:
         ci_instance = CloudInterface()
     return ci_instance
+
+
+def directory_summary_hash(dirname: str):
+    """ Computes hash of directory summary"""
+    hash_func = hashlib.md5
+
+    if not os.path.isdir(dirname):
+        raise TypeError("{} is not a directory.".format(dirname))
+
+    hash_values = []
+    for root, dirs, _ in os.walk(dirname, topdown=True, followlinks=False):
```

Comment:
We decided to go with the folders directly in the snapshot folder, not recursively